tests/unit/storage/data/DummyService.js

```diff
@@ -3,6 +3,8 @@ const { EventEmitter } = require('events');
 
 const DummyObjectStream = require('./DummyObjectStream');
 const { parseRange } = require('../../../../lib/network/http/utils');
+const assert = require('assert');
+const errors = require('../../../../errors/arsenalErrors.json');
```

Comment:
I don't think we import the json here, you should import lib/errors which will turn the JSON into objects. It is a little confusing.

Reply:
I need it for line 140 and line 160 to set the error to `NoSuchKey`.

Reply:
My bad I understand what you mean, ok I will update it and set the error to an actual `ArsenalError` object

Reply:
done